### PR TITLE
#16715 Log one line error from dotcms

### PR DESCRIPTION
**Pull request by dsilvam**

### CODE REVIEW

**Review by nollymar — APPROVED** [review]

File: dotCMS/src/main/java/com/dotmarketing/portlets/folders/exception/InvalidFolderNameException.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.dotmarketing.portlets.folders.exception;
+
+import com.dotmarketing.exception.DotDataException;
+
+public class InvalidFolderNameException extends DotDataException {
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotmarketing/portlets/folders/exception/InvalidFolderNameException.java]: doc?

File: dotCMS/src/main/java/com/dotmarketing/webdav/HostResourceImpl.java [hunk 2]
```diff
@@ -266,8 +269,13 @@ public CollectionResource createCollection(String newName) throws DotRuntimeExce
 			FolderResourceImpl fr = new FolderResourceImpl(f, path + newName + "/");
 			return fr;
 		} catch (Exception e) {
-			Logger.error(this, e.getMessage(), e);
-			throw new DotRuntimeException(e.getMessage(), e);
+			if(e instanceof InvalidFolderNameException) {
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotmarketing/webdav/HostResourceImpl.java]:
```suggestion
} catch (InvalidFolderNameException e) {
   ....
} catch (Exception e) {
 ...
}
```

**Review by freddyucv — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/java/com/dotmarketing/db/DotCMSInitDb.java [hunk 3]
```diff
@@ -113,11 +113,11 @@ private static void loadStarterSite(PrintWriter pw) throws IOException{
...
 		 }
-		
-		ImportExportUtil ieu = new ImportExportUtil();
-		if(ieu.validateZipFile(starterZip)){
-		    Try.run((()->ieu.doImport(pw))).onFailure(e-> new DotRuntimeException(e));
 
-		}
+		ImportExportUtil ieu = new ImportExportUtil();
+		if(ieu.validateZipFile(starterZip))
+			Try.run((() -> ieu.doImport(pw))).onFailure(e -> {
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/db/DotCMSInitDb.java]: Codacy Issue found: [This statement should have braces](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4610406)

**Review by freddyucv — APPROVED** [review]

File: dotCMS/src/main/java/com/dotmarketing/db/DotCMSInitDb.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/db/DotCMSInitDb.java]: Codacy Issue found: [This statement should have braces](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4610406)

File: dotCMS/src/main/java/com/dotmarketing/webdav/HostResourceImpl.java [hunk 4]
```diff
@@ -265,6 +268,9 @@ public CollectionResource createCollection(String newName) throws DotRuntimeExce
 			Folder f = dotDavHelper.createFolder(path + newName, user);
 			FolderResourceImpl fr = new FolderResourceImpl(f, path + newName + "/");
 			return fr;
+		} catch(InvalidFolderNameException e) {
+			Logger.warnAndDebug(HostResourceImpl.class, e.getMessage(), e);
+			throw new DotRuntimeException(e.getMessage());
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotmarketing/webdav/HostResourceImpl.java]: do not loss the original exception 
```
throw new DotRuntimeException(e.getMessage(), e);
```

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/main/java/com/dotmarketing/webdav/HostResourceImpl.java (hunk 4 above)
Comment (dsilvam) [reply dotCMS/src/main/java/com/dotmarketing/webdav/HostResourceImpl.java]: that's exactly what we want here

**Pull request merged**